### DISCUSSION TITLE
0.0.1

### DISCUSSION THREAD

**Discussion by JarvyJ** [JarvyJ/pislide-os#12] [Announcements]
## The first release!
We now have system images for a couple of Pi revisions and have tested on a Pi 3/4. If you try it out on a Pi 2, 5, or 0v2 and it works well let us know!

## Features
- Support for a large variety of image formats: JPG, PNG, WEBP, AVIF, JXL, HEIF, HEIC, SVG, BMP, TIFF, and QOI
- Config based slideshow settings
- Easy to load images, just drag-n-drop to an SD Card
- Works offline just fine

**Full Changelog**: https://github.com/JarvyJ/pislide-os/commits/0.0.1

<hr /><em>This discussion was created from the release <a href='https://github.com/JarvyJ/pislide-os/releases/tag/0.0.1'>0.0.1</a>.</em>

**Comment by adamgenesiseden-dev** [comment]
I downloaded the RP4 version and flashed it to a 64GB microSD card.
I put it into my RP4 case and it loads awhile and then drops to the console screen.
I put the SD card back into my Windows and it shows a new directory PHOTOS.
I go into PHOTOS and create a MOVIEPOSTERS folder.  I copy all my posters to the folder.
I edit the .txt file to show MOVIEPOSTERS.
Put the SD card back into RP4 and it returns to the console screen.  Doesn't boot PiSlide at all.
HELP!

**Reply by adamgenesiseden-dev** [reply]
<img width="971" height="699" alt="crash" src="https://github.com/user-attachments/assets/9098887c-ad1d-416a-b43e-93876a0edb69" />
